PaperlessNgx/PaperlessNgx.php

```diff
@@ -14,7 +14,7 @@ public function getRequestAttrs()
 
 		$attrs["headers"] = [
 			"Accept" => "application/json",
-			"Authorization" => $apikey,
+			"Authorization" => "Token " .  $apikey,
```

Comment:
Sure it's Token and not something like

        Authorization: Bearer <token>

? 🤔

Reply:
Yes its Token, strange but ok ...
https://docs.paperless-ngx.com/api/